src/site/antora/modules/ROOT/examples/migrate-from-logback/MigrateFromLogback.java

```diff
@@ -0,0 +1,28 @@
+package example;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+class MigrateFromLogback {
+    private static Logger logger = LogManager.getLogger();
+
+    doLogWrong() {
+        try {
+            // do something
+            // tag::wrong[]
+        } catch (Exception e) {
```

Comment:
Could you, please, please, please use self-explanatory variable names in the docs? For instance, `error`, `failure`, etc.

Reply:
Fixed in 42b5ebbb41b4b23f665d395833a10c431a2a602f.